src/component/common/calender.css

```
@@ -0,0 +1,69 @@
+}
+
+.weekday-cell {
+  font-size: 12px;
+  text-align: center;
+  color: #666;
+  font-weight: 500;
+  padding: 2px 0;
+}
+
+.dates-grid {
+  height: 25pxc;
```

Biome (1.9.4)

[error] Unexpected unknown unit: pxc

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)